index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=, initial-scale=1.0" />
+    <title>Portfolio|Dahir-Mohamed</title>
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="./assets/style.css" />
+  </head>
+
+  <body>
+    <!-- The logo and nav-links are nested in a header tag  -->
+
+    <header class="header">
+      <h1 class="logo">PORTFOLIO</h1>
+
+      <nav class="nav-bar-link">
+        <div class="nav-link"><a class="link" href="#about">About Me</a></div>
+        <div class="nav-link">
+          <a class="link" href="#projects">Projects</a>
+        </div>
+        <div class="nav-link">
+          <a class="link" href="#contact">Contact Me</a>
+        </div>
+      </nav>
+    </header>
+
+    <!-- The h1 tag containing my name is nested in a div tag that also contains a background img   -->
+
+    <div class="banner">
+      <h1 class="name">Dahir A Mohamed</h1>
+    </div>
```

Comment:
This `h1` overflows when it's on a mobile view. Take a look at the stlying for it so the whole text is viewable.